hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -767,6 +767,29 @@ public void testSnapDiffMultipleBuckets() throws Exception {
     Assert.assertEquals(1, diff1.getDiffList().size());
   }
 
+  @Test
+  public void testListSnapshotDiffWithInvalidParameters()
+      throws Exception {
+    String volume = "vol-" + RandomStringUtils.randomNumeric(5);
+    String bucket = "buck-" + RandomStringUtils.randomNumeric(5);
+
+    String errorMessage = "Provided volume name " + volume +
+        " or bucket name " + bucket + " doesn't exist";
+    // List SnapshotDiff jobs for non existent volume and bucket.
+    LambdaTestUtils.intercept(OMException.class,
```

Comment:
nit: Use `Assert.assertThrows()` directly.

```suggestion
    OMException omException = assertThrows(OMException.class,
        () -> store.listSnapshotDiffJobs(volume, bucket, "", true));
   assertEquals(errorMessage, omException.getMessage());
```